google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -429,15 +430,23 @@ class Organization(Resource):
     def fetch(cls, client, resource_key):
         """Get Organization
 
+        Saves ApiExecutionErrors as warnings.
+
         Args:
             client (object): GCP API client
             resource_key (str): resource key to fetch
 
         Returns:
             Organization: created Organization
         """
-        data = client.fetch_organization(resource_key)
-        return FACTORIES['organization'].create_new(data, root=True)
+        try:
+            data = client.fetch_organization(resource_key)
+            return FACTORIES['organization'].create_new(data, root=True)
+        except api_errors.ApiExecutionError as e:
```

Comment:
Can we please add a warning log line here, so that this will show up in stackdriver instead of just the database?